compiler/src/main/java/com/squareup/anvil/compiler/codegen/ContributesSubcomponentHandlerGenerator.kt

```diff
@@ -186,7 +186,7 @@ internal class ContributesSubcomponentHandlerGenerator(
 
     val newContributions = contributions
       .flatMap { contribution ->
-        newTriggers
+        triggers
```

Comment:
I actually made this change to fix the referenced bug, but it introduced regressions and wasn't the proper fix.